src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -1378,24 +1378,27 @@ void uct_dc_mlx5_ep_handle_failure(uct_dc_mlx5_ep_t *ep, void *arg,
     ucs_arbiter_group_t *group;
     uint8_t pool_index;
 
+    ucs_debug("handle failure iface: %p, dci[%d]=%x, status: %s", iface,
+              dci_index, txwq->super.qp_num, ucs_status_string(ep_status));
```

Comment:
plz update trace here as well, like in `uct_dc_mlx5_iface_reset_dci`

Reply:
updated